test/testPlugin.js

```diff
@@ -47,28 +46,30 @@ export const testPlugin = {
         test.done();
     },
 
-    testPluginGetType: function(test) {
-        test.expect(2);
+    testPluginGetRulesDefault: function(test) {
+        test.expect(3);
 
         const plugin = new MockPlugin();
 
         test.ok(plugin);
 
-        test.equal(plugin.getType(), "resource");
+        const rules = plugin.getRules();
+        test.ok(rules);
+        test.equal(rules.length, 0);
```

Comment:
How about adding a test case for when the plugin has rules?

Reply:
These tests test the superclass/abstract class with default methods that do not do much. The test for when real rules are returned are in the i18nlint tool itself.

Reply:
oh I see.